metricshub-doc/src/site/markdown/configuration/configure-agent.md

```diff
@@ -872,3 +882,31 @@ Timeouts, durations and periods are specified with the below format:
 | m    | minutes                         | 90m, 1m15s       |
 | h    | hours                           | 1h, 1h30m        |
 | d    | days (based on a 24-hour day)   | 1d               |
+
+## Customise the hostname
+
+By default, the `host.name` attribute is used as:
+1. The hostname or IP address of the resource
+2. A hostname on each metric in Prometheus.
+
+If the `hostname` parameter is specified in the protocol configuration, it overrides the `host.name` attribute for client requests. The `host.name` will be used only as a metric hostname in Prometheus.

```

Comment:
If the `hostname` parameter is specified in the protocol configuration, it overrides the `host.name` attribute for client requests. The `host.name` will be used only as a metric attribute.